compute_tools/src/http/openapi_spec.yaml

```diff
@@ -254,10 +254,28 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/GenericError"
+
+  /check_config:
```

Comment:
I don't see the benefits of adding extra API method for this check.

Cons:
- this adds another roundtrip from cplane to compute (first check config, then run configure)
- this requires extra changes in cplane (call check_config)

What was your idea here?

You can call `check_config()` in `/configure` handler

Reply:
The `check_config()` call has been added to the `/configure` handler yes, the idea is to allow for “check only” either in unit testing, or maybe other production code paths that I don't know about yet. IOW, flexibility.

Reply:
New API removed.